monitoring.go

```diff
@@ -347,6 +350,14 @@ func (sc *snowflakeConn) blockOnRunningQuery(
 				message = fmt.Sprintf("%s: (failed to parse original code: %s: %s)", message, resp.Code, err.Error())
 			}
 		}
+		if code == -1 {
+			ok, deadline := ctx.Deadline()
+			logger.WithContext(ctx).Errorf("Deadline: %v, ok: %v", deadline, ok)
+			logger.WithContext(ctx).Errorf("response: %v, error: %v", resp, err)
```

Comment:
I'd be a little careful about just logging `resp`, since it may contain customer data. If possible I'd suggest just logging specific fields, or otherwise figuring out specific properties you want to check for.

Reply:
Also idk if there's a way to get our request IDs into here (or maybe we could use snowflake request IDs?) but just FYI I don't think these log lines will have any context attached to them to link with actual requests.

Reply:
Ok thank you, I will just log what I need from the response, and I will log the snowflake queryID which I also have in the request table